bitacross-worker/bitacross/core/bc-musig2-ceremony/src/lib.rs

```diff
@@ -392,11 +397,22 @@ impl<AK: AccessKey<KeyType = SchnorrPair>> MuSig2Ceremony<AK> {
 					info!("Message {:?}", message);
 
 					info!("Verification result: ");
-					match verify_single(self.agg_key.unwrap(), signature, message) {
-						Ok(_) => info!("OK!"),
-						Err(_) => info!("NOK!"),
+					let result = match verify_single(self.agg_key.unwrap(), signature, message) {
+						Ok(_) => {
+							info!("OK!");
```

Comment:
maybe a bit picky, but can we push some context info here for log, as anyway we output log.
Same for the below "NOK".

Reply:
i'll remove it